showcase/app/routes/page-components/advanced-table.ts

```diff
@@ -6,14 +6,17 @@
 import Route from '@ember/routing/route';
 
 import { DENSITIES } from '@hashicorp/design-system-components/components/hds/advanced-table/index';
-import type {
-  HdsBadgeColors,
-  HdsBadgeTypes,
-} from '@hashicorp/design-system-components/components/hds/badge/types';
-import type { HdsIconSignature } from '@hashicorp/design-system-components/components/hds/icon/index';
 
 import type { ModelFrom } from 'showcase/utils/ModelFromRoute';
 
+import clusterData from 'showcase/mocks/cluster-data';
+import folkMusicData from 'showcase/mocks/folk-music-data';
+import userWithManyColumnsData from 'showcase/mocks/user-with-more-columns-data';
```

Comment:
```suggestion
import usersWithMoreColumns from 'showcase/mocks/user-with-more-columns-data';
```
[Issue] There's a naming mismatch here and with the model below where the data says "more" and here it says "many". Not sure if it'd be easier to update the data file or the imports, but some alignment should happen.